index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <h4>My Logo</h4>
+        <nav>
+            <a href="">
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
```

Comment:
Good job for moving the h4 outside the nav 👍 
But you forgot to move the link too.

Please update your code as follow to correct it:

```
<header>
  <a href="">
       <h4>My Logo</h4>
   </a>
        <nav>
            <img src="./img/menu.png" alt="web menu graphics">
        </nav>
</header>
```